ArduCopter/switches.cpp

```diff
@@ -370,12 +370,14 @@ void RC_Channel_Copter::do_aux_function(const aux_func_t ch_option, const aux_sw
             switch (ch_flag) {
                 case LOW:
                     copter.landinggear.set_position(AP_LandingGear::LandingGear_Deploy);
+                    copter.gear_sw_up = false;
```

Comment:
Some time in the future I'll make a `bool rc().get_aux_value(const aux_func_t ch_option, aux_switch_pos_t &ch_flag)` method.  This will save having to keep the switch state around like this.

Reply:
I was actually just thinking "I wonder if peter made something to do this with more efficiently". 👍